src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -44,7 +44,164 @@ paths:
+  '/eros/{eroId}/applications/{applicationId}':
+    parameters:
+      - name: eroId
+        description: The ID of the Electoral Registration Office responsible for the application.
+        schema:
+          type: string
+        in: path
+        required: true
+      - name: applicationId
+        description: The identifier of the Voter Authority Certificate to retrieve.
+        schema:
+          type: string
+        in: path
+        required: true
+    options:
+      summary: CORS support
+      description: |
+        Enable CORS by returning correct headers
+      tags:
+        - Single Certificate
+      responses:
+        200:
+          description: Default response for CORS method
+          headers:
+            Access-Control-Allow-Origin:
+              schema:
+                type: string
+            Access-Control-Allow-Methods:
+              schema:
+                type: string
+            Access-Control-Allow-Headers:
+              schema:
+                type: string
+          content: { }
+      x-amazon-apigateway-integration:
+        type: mock
+        requestTemplates:
+          application/json: |
+            {
+              "statusCode" : 200
+            }
+        responses:
+          default:
+            statusCode: "200"
+            responseParameters:
+              method.response.header.Access-Control-Allow-Headers: '''Content-Type,X-Amz-Date,Authorization,X-Api-Key'''
+              method.response.header.Access-Control-Allow-Methods: '''*'''
+              method.response.header.Access-Control-Allow-Origin: '''*'''
+            responseTemplates:
+              application/json: |
+                {}
+    get:
+      summary: Returns a single certificate
+      description: Returns a single certificate
+      tags:
+        - Single Certificate
+      responses:
+        '200':
+          $ref: '#/components/responses/Certificate'
+        '404':
+          description: Not Found
+      operationId: get-certificate-by-application-id
+      security:
+        - eroUserCognitoUserPoolAuthorizer: [ ]
+      x-amazon-apigateway-integration:
+        type: HTTP_PROXY
+        uri: '${base_uri}/eros/{eroId}/applications/{applicationId}'
+        requestParameters:
+          integration.request.path.eroId: method.request.path.eroId
+          integration.request.path.applicationId: method.request.path.applicationId
+        responseParameters:
+          method.response.header.Access-Control-Allow-Headers: '''Content-Type,X-Amz-Date,Authorization,X-Api-Key'''
+          method.response.header.Access-Control-Allow-Methods: '''*'''
+          method.response.header.Access-Control-Allow-Origin: '''*'''
+        connectionType: VPC_LINK
+        connectionId: '${vpc_connection_id}'
+        httpMethod: GET
+
 components:
+  #
+  # Schema and Enum Definitions
+  # --------------------------------------------------------------------------------
+  schemas:
+    CertificateResponse:
```

Comment:
I was wondering if this was just a summary - the print request that is included seems to be a brief summary - so don't know if we will have in the future objects that include full details.

Reply:
Makes sense to change it to summary, if there will be an endpoint in the future with full details. @NathanRussellValtech , what do you think?

Reply:
Agreed. I'm not 100% what our future API requirements will be. We know we will have a different API soon that allows to search for certs and their print requests. Whether this is the same response object type (a collection of them), I don't know at this point.

What I do think though is that what we are returning here is not the full Certificate with all of it's print responses etc. Whether we'll ever have a need to? I honestly don't know, but I think @Neil-Massey is right in that this is more like a summary that in turn contains a list of summaries of the print requests (that for the most part is the last status of each print request)

Reply:
@NathanRussellValtech , @Neil-Massey , thanks for the suggestion. 
I have renamed Certificate to CertificateSummary accordingly.